### ISSUE TITLE
ValueError while training

### ISSUE THREAD

**Issue by godfatherlmh** [vingtfranc/LoLAnalyzer#8]
Sorry I'm new to github. Not sure if I'm supposed to open an issue for this or if there is a way to just message you for help.

After the data shuffling, it gave me a ValueError while training. 

![image](https://user-images.githubusercontent.com/48610302/55709096-22bd6d80-5a1a-11e9-846b-f61dd4dad3c6.png)
I restarted over RunAll.py but I still have the same error.

**Comment by godfatherlmh** [comment]
I have solved it by installing an old version of keras. Sorry for the trouble.

**Comment by VRichardJP** [comment]
Hi,

Can you give me the version you've installed?

I see you have a small amount of data. You can change the amount of files used for training by changing `keep_for_testing` value (should be a prime number around 10% of the total number of data).

By the way, I would be more than happy if you can create a new pull request with the changes you have made to the code (I'm sorry I don't have much time to look at it myself)
